src/algebra/lattice_ordered_group.lean

```diff
@@ -550,4 +550,38 @@ begin
     apply lattice_ordered_comm_group.inv_le_abs, }
 end
 
+/--
+Let `α` be a lattice ordered commutative group and let `a` and `b` be elements in `α`. Then,
+`|a - b|` the absolute value of `a - b`, is equal to `|b - a|`, the absolute value of `b - a`.
+-/
+@[to_additive]
+lemma abs_inv_invariant (a b : α) : |a/b| = |b/a| :=
```

Comment:
The docstring by default is for the multiplicative version. At any rate, this docstring feels too verbose to me:
```suggestion
/-- In a lattice ordered commutative group, division commutes within `abs`. -/
@[to_additive /-" In an lattice ordered additive commutative group, subtraction commutes within `abs`. -/]
lemma abs_div_comm (a b : α) : |a/b| = |b/a| :=
```